runtime/doc/news.txt

```diff
@@ -15,6 +15,13 @@ BREAKING CHANGES                                                *news-breaking*
 
 The following changes may require adaptations in user config or plugins.
 
+• In some cases, the cursor in Nvim would blink even without configuring
+  'guicursor' as mentioned in |cursor-blinking|. This was a bug that has now
+  been fixed. If your cursor has stopped blinking, add the following (or
+  similar, adapted to user preference) to your |config| file: >vim
+
+    set guicursor+=n:blinkon100-blinkoff100
```

Comment:
The values of `blinkon` and `blinkoff` don't appear to actually be interpreted as intervals in milliseconds. As far as I can tell, `blinkon` is actually a boolean flag, which means the following is sufficient:

```vim
set guicursor+=n:blinkon1
```

But this contradicts the help text in `:h cursor-blinking`. cc @justinmk

Reply:
I suppose that happens because you can ask the terminal to make the cursor blink, but not set a frequency to it (unlike in a custom UI).

Reply:
Right, my suspicion is that the "interval in milliseconds" thing is for external GUIs, but not for the TUI.

Reply:
yes, exactly. the value is there for GUIs to interpret. the TUI can't do anything with it.